backend/src/main/java/com/bakdata/conquery/apiv1/frontend/FESelect.java

```diff
@@ -14,4 +15,6 @@
 	private String label;
 	private String description;
 	private ResultType resultType;
+	@JsonProperty("default")
+	private Boolean isDefault;
```

Comment:
Ist `@JsonProperty("default")` hier notwendig?

Reply:
Macht Jackson das `is` nicht automatisch?

Reply:
das wäre Lombok denke was das setzt, aber ich kann das Feld selber nicht default nennen. So ist das denke ich sauberer.